### PR TITLE
Docs for multiple line plots.

### PR DESCRIPTION
**Pull request by dcherian**
As requested, I've added a section to the 1D plotting documentation.

I don't like the title. but couldn't think of a better one...

### CODE REVIEW

File: doc/plotting.rst [hunk 1]
```diff
@@ -174,14 +174,42 @@ entire figure (as for matplotlib's ``figsize`` argument).
...
 
+
+Multiple lines showing variation along a dimension
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to make line plots of two-dimensional data by calling ``xarray.plot.line()`` with appropriate arguments. Consider the 3D variable ``air``
+
+.. ipython:: python
+
+    air
```
Comment (shoyer) [inline doc/plotting.rst]: We use `air` as a running example on this page. I don't think we need to print it again here.

File: doc/plotting.rst [hunk 2]
```diff
@@ -174,14 +174,42 @@ entire figure (as for matplotlib's ``figsize`` argument).
...
+It is required to explicitly specify either
+
+1. ``x``: the dimension to be used for the x-axis, or
+2. ``hue``: the dimension you want to represent by multiple lines.
+
+Thus, we can make the previous plot by specifying ``hue='lat'`` instead of ``x='time'``. If required, the automatic legend can be turned off using ``add_legend=False``.
+
+.. ipython:: python
+
+    @savefig plotting_example_multiple_lines_hue_kwarg.png
```
Comment (shoyer) [inline doc/plotting.rst]: I would omit this example. `add_legend=False` doesn't need an example to make it clear what it does, and plots take up lots of space in the docs.

Comment (dcherian) [reply doc/plotting.rst]: It also illustrates hue, but we could just leave that as a text description.

Is there eat you would prefer?

Comment (shoyer) [reply doc/plotting.rst]: I'm OK with it in the text description.

**Comment by fmaussion** [comment]
Ups, I realized that was a bit fast of a merge. I hope everyone agrees.

Thanks @dcherian !

**Comment by fmaussion** [comment]
See it rendered [here](http://xarray.pydata.org/en/latest/plotting.html#multiple-lines-showing-variation-along-a-dimension)

**Pull request merged**